soc/arm/nordic_nrf/Kconfig

```diff
@@ -49,6 +49,7 @@ config NRF_ACL_FLASH_REGION_SIZE
 config NFCT_PINS_AS_GPIOS
 	bool "NFCT pins as GPIOs"
```

Comment:
would add `[DEPRECATED]` or similar here so it's obvious

Reply:
added